sdk/dotnet/Pulumi/Pulumi.csproj

```diff
@@ -30,26 +30,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="3.1.15" />
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Ben.Demystifier" Version="0.4.1" />
```

Comment:
Also curious about their stance on adding additional dependencies.

Reply:
Reading what this does, it improves stack traces, and seems reasonably supported/popular; I think stack traces in Pulumi do need some care, so I'd take this. Asking @mikhailshilkov for a second opinion..

Reply:
I'm also using it to improve the type display name in [Output\<T\>.ctor](https://github.com/pulumi/pulumi/blob/cb539c58d2fad759d1f20f8923fce877ae3b79a8/sdk/dotnet/Pulumi/Core/Output.cs#L147).